monai/data/meta_obj.py

```diff
@@ -177,6 +139,7 @@ def _copy_meta(self, input_objs: list[MetaObj]) -> None:
         id_in = id(input_objs[0]) if len(input_objs) > 0 else None
         deep_copy = id(self) != id_in
         self._copy_attr("meta", input_objs, self.get_default_meta, deep_copy)
+        self._copy_attr("applied_operations", input_objs, self.get_default_applied_operations, deep_copy)
```

Comment:
Should we add this `applied_operations` to the `TraceKeys`?
https://github.com/Project-MONAI/MONAI/blob/dev/monai/utils/enums.py#L227
Or create another class for `MetaObj` keys?

Thanks.